app/controllers/users_controller.rb

```diff
@@ -15,10 +15,23 @@ def show
   # GET /users/new
   def new
     @user = User.new
+    a = 1 + 2
+    if a > 3
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.